stacks_queues/stack.py

```diff
@@ -12,7 +13,8 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)
+        return None
```

Comment:
The LinkedList implementation provided is doubly linked with a tail pointer, which makes adding at the end efficient. If this were singly-linked we would want to use operations at the head instead.

We also don't need to explicitly return `None` (this is the default behavior when there's no return).